CRM/Friend/Form.php

```diff
@@ -45,14 +45,21 @@ class CRM_Friend_Form extends CRM_Core_Form {
   const NUM_OPTION = 3;
 
   /**
-   * The id of the entity that we are proceessing.
+   * The id of the entity that we are processing.
    *
    * @var int
    */
   protected $_entityId;
 
   /**
-   * The table name of the entity that we are proceessing.
+   * Tell a friend id in db.
+   *
+   * @var int
+   */
+  public $_friendId;
```

Comment:
will it be protected instead of public?

Reply:
It's used in public static function buildFriendForm($form) which is called from two other classes (Contribute and Event) so it needs to be public unless we do a lot more work, at which point we would risk breaking things :-)